prow/kube/client.go

```diff
@@ -214,8 +214,6 @@ func (c *Client) requestRetry(r *request) ([]byte, error) {
 		return nil, NewConflictError(fmt.Errorf("body: %s", string(rb)))
 	} else if resp.StatusCode == 422 {
 		return nil, NewUnprocessableEntityError(fmt.Errorf("body: %s", string(rb)))
-	} else if resp.StatusCode == 404 {
```

Comment:
Why did you remove this?

Reply:
404 is already handled just above. Sadly GitHub diff doesn't show it. :-)